src/app/containers/Header/index.jsx

```diff
@@ -49,11 +51,12 @@ const HeaderContainer = () => {
 
   return (
     <header role="banner" lang={serviceLang}>
-      <ConsentBanner />
+      <ConsentBanner brandRef={brandRef} />
```

Comment:
```suggestion
      <ConsentBanner onDismissFocusRef={brandRef} />
```
Just showing here the effect of the naming change from above, it shows that 'on dismiss we focus on the brand' rather than just passing in a reference to the brand for an unknown reason.